src/main/java/net/machpi/runelite/influxdb/InfluxDbPlugin.java

```diff
@@ -60,23 +71,57 @@ InfluxDbConfig getConfig(ConfigManager configManager) {
      */
     private final ScheduledExecutorService executor = new ExecutorServiceExceptionLogger(Executors.newSingleThreadScheduledExecutor());
 
+    private ActivityState activityState;
+    private boolean loginFlag;
+    private Map<Skill, Integer> skillExp = new HashMap<>();
```

Comment:
Not really a fan of this method of ignoring the initial XP change.  I'd prefer just a boolean that you unset whenever logging in and set the first tick after being logged in.